virttest/libvirt_xml/devices/memory.py

```diff
@@ -113,6 +114,11 @@ def __init__(self, virsh_instance=base.base.virsh):
                                          'virsh_instance': virsh_instance})
             accessors.XMLElementBool('readonly', self, parent_xpath='/',
                                      tag_name='readonly')
+            accessors.XMLElementNest('address', self, parent_xpath='/',
+                                     tag_name='address', subclass=Memory().Address,
```

Comment:
```suggestion
                                     tag_name='address', subclass=Memory.Address,
```
Would this way also work? Therefore we might not need to instantiate `Memory` class.

Reply:
Yes ,  it works. Thanks haijiao : D